kairon/shared/data/data_objects.py

```diff
@@ -479,8 +480,8 @@ class StoryEvents(EmbeddedDocument):
     def validate(self, clean=True):
         if clean:
             self.clean()
-        if not Utility.check_empty_string(self.value) and self.type != 'slot':
```

Comment:
just reverse it: self.type != 'slot' and not Utility.check_empty_string(self.value)